src/pivot-table/hooks/use-head-cell-dim.ts

```diff
@@ -8,7 +8,10 @@ interface UseHeadCellDim {
 export const useHeadCellDim = ({ interactions }: UseHeadCellDim) => {
   const [open, setOpen] = useState(false);
 
-  const handleOpenMenu = () => interactions.active && setOpen(true);
+  const handleOpenMenu = (evt: React.MouseEvent) =>
+    interactions.active &&
+    !(evt.target as HTMLElement)?.className.includes("sn-pivot-table-column-adjuster") &&
```

Comment:
Potential refactor to have this as a util since we use it in multiple places now

Reply:
Should at least make a constant with the class name

Reply:
I'll leave this for the next PR. Will do some small refactoring